ui/app/AppLayouts/Communities/views/EditPermissionView.qml

```diff
@@ -199,11 +201,23 @@ StatusScrollView {
             addButton.visible: count < d.maxHoldingsItems &&
                                whoHoldsSwitch.checked
 
+            SortFilterProxyModel {
```

Comment:
WDYT if instead of adding the filter here you add it in `PermissionsSettingsPanel` line `188` since the `collectiblesModel` used inside `EditPermissionView` must be the filtered one always? Then you can forget about replacing the `nonOwnerCollectibles` object in line _220_ and _257_ here. There are also 2 more calls to `collectiblesModel` in _299_ and _323_ that should be addressed if you opt for the current solution!

Reply:
Thanks, I had made a component analysis before making the changes, and I saw that PermissionsView would also be impacted if I update the Filter at the `PermissionsSettingsPanel` root level. Wouldn't it be clearer to the future reader that the filter only applies to `EditPermissionView` ?

```qml
CommunitySettingsView {
	required property var permissionsModel
	required property var assetsModel
	required property var collectiblesModel
	required property var channelsModel
	
	PermissionsSettingsPanel {
		required property var permissionsModel
    		required property var assetsModel
    		required property var collectiblesModel
		required property var channelsModel
		
		PermissionsView {
			required property var permissionsModel
    			required property var assetsModel
    			required property var collectiblesModel
    			required property var channelsModel
    			
    			PermissionItem {
				property var holdingsListModel = HoldingsSelectionModel {
                			sourceModel: model.holdingsListModel
                			assetsModel: root.assetsModel
                			collectiblesModel: root.collectiblesModel
            			}
			    	property var channelsListModel
    			}
		}
		
		EditPermissionView {		
			required property var assetsModel
			required property var collectiblesModel
			required property var channelsModel
		}
	}
}
```

Reply:
After second thought, yes your idea is simplication how to inject the collectiblesModel to the `EditPermissionView`. Updated thanks